django/utils/feedgenerator.py

```diff
@@ -57,6 +58,40 @@ def get_tag_uri(url, date):
     return "tag:%s%s:%s/%s" % (bits.hostname, d, bits.path, bits.fragment)
 
 
+NOT_PROVIDED = object()
+
+
+class Stylesheet:
+    """An RSS stylesheet"""
+
+    MIMETYPE_OVERRIDES = {
+        # The official mimetype for XSLT files is technically `application/xslt+xml`
+        # but as of 2024 almost no browser support that.
+        "application/xslt+xml": "text/xsl",
+    }
+
+    def __init__(self, url, mimetype=NOT_PROVIDED, media="screen"):
+        self.url = iri_to_uri(url)
+
+        if mimetype is NOT_PROVIDED:
+            mimetype, _ = mimetypes.guess_type(self.url)
```

Comment:
This is the main thing that causes the break in the windows tests I think
mimetype is "guessed" as xml on windows

Reply:
:very_happy_face: nice work @bmispelon - tricky one! 🥳